step-templates/octopus-apply-octoterra-module-s3.json

```diff
@@ -126,7 +126,7 @@
       "Name": "OctoterraApply.AWS.S3.BucketKey",
       "Label": "AWS S3 Bucket Key",
       "HelpText": "The S3 file used to hold the Terraform state. See the [Terraform documentation](https://developer.hashicorp.com/terraform/language/settings/backends/s3) for details on using S3 as a backend. The combination of the workspace name and this key must be unique. \n\nThe default value is the name of the project: `#{Octopus.Project.Name | Replace \"[^A-Za-z0-9]\" \"_\"}`.",
-      "DefaultValue": "#{Octopus.Project.Name | Replace \"[^A-Za-z0-9]\" \"_\"}",
+      "DefaultValue": "Project_#{Octopus.Project.Name | Replace \"[^A-Za-z0-9]\" \"_\"}",
```

Comment:
Help Text seems to specify a different value than the DefaultValue attribute, DefaultValue is prefixing `Project_`

Reply:
:+1: I'll update that text.